gt/main.c

```diff
@@ -984,6 +984,8 @@ gt_proc(void *arg)
 	struct rte_ip_frag_death_row death_row;
 	death_row.cnt = 0;
 
+	uint16_t gt_max_pkt_burst = gt_conf->gt_max_pkt_burst;
```

Comment:
The line `death_row.cnt = 0;` above is the first instruction before variable declarations, so move the declaration of variable `gt_max_pkt_burst` above.